lib/api/fragments/global.js

```diff
@@ -5,10 +5,15 @@ fragment pageTreeFragment on EntryInterface {
   id
   title
   uri
-  children(type: ["not", "educatorPages"]) {
+  children(isVisible: true) {
```

Comment:
filtering on `isVisible` to only get the nav items that ought to be there.  Also deleted the filter on `educatorPages` because I couldn't think of any reason to not allow them in the nav to begin with.  Not sure what the previous rationale might have been for that exclusion but.....I think it's fine.  Access to them is restricted but I think there's no harm in allowing someone to reach those pages.